ignite/contrib/handlers/clearml_logger.py

```diff
@@ -178,6 +178,24 @@ def bypass_mode(cls) -> bool:
         """
         return getattr(cls, "_bypass", bool(os.environ.get("CI")))
 
+    def __getattr__(self, attr: Any) -> Any:
+        """
+        Calls the corresponding method of ``ClearML.Logger``.
```

Comment:
```suggestion
        Calls the corresponding method of ``clearml.Logger``.
```

Class is exactly `clearml.Logger`: https://github.com/allegroai/clearml/blob/master/clearml/__init__.py

Reply:
Fixed.